includes/admin/post-types/meta-boxes/class.llms.meta.box.course.options.php

```diff
@@ -350,6 +350,17 @@ public function get_fields() {
 						'label'            => __( 'Capacity Reached Message', 'lifterlms' ),
 						'type'             => 'text',
 					),
+					// Require sequential completetion
```

Comment:
This method & class needs a `@since` tag addition

Reply:
Done